### ISSUE TITLE
sourceMap sourceRoot information is missing

### ISSUE THREAD

**Issue by zsl99a** [egoist/rollup-plugin-postcss#49]
Ideal result
```
{
  "version": 3,
  "sources": [
    "src/routes/admin/admin.css"
  ],
  "names": [],
  "mappings": "AAAA;EACE,YAAY;EACZ,aAAa;CACd;;AAED;EACE,eAAe;CAChB",
  "file": "bundle.css",
  "sourcesContent": [
    ".main {\n  width: 100%;\n  height: 100%;\n}\n\n.sider {\n  overflow: auto;\n}\n"
  ]
}
```

actual results
```
{
  "version": 3,
  "sources": [
    "admin.css"
  ],
  "names": [],
  "mappings": "AAAA;EACE,YAAY;EACZ,aAAa;CACd;;AAED;EACE,eAAe;CAChB",
  "file": "bundle.css",
  "sourcesContent": [
    ".main {\n  width: 100%;\n  height: 100%;\n}\n\n.sider {\n  overflow: auto;\n}\n"
  ]
}
```

**Comment by egoist** [comment]
should be fixed in `v1.0`

**Comment by NickolasBoyer** [comment]
Version `2.0.3` has this problem, `src/test/main.css` is instead generated as `main.css`.

Example:
```json
{
  "version": 3,
  "sources": ["main.css"],
  "names": [],
  "mappings": "AAAA;EACE,UAAU;AACZ",
  "file": "main.css",
  "sourcesContent": ["body {\n  color: red;\n}"]
}
```
